config/locales/es.yml

```diff
@@ -13,6 +13,18 @@ es:
       return_to_profile: Volver al Perfil
       my_clients: Mis Clientes
       my_profile: Mi Perfil
+    zip_codes:
+      already_taken: "%{zip_code} ya está enrutado a <a href=%{vita_partner_path}>%{vita_partner_name}</a>."
+      already_applied: "%{zip_code} ya está enrutado a este socio."
+      not_found: Código postal no encontrado. Intentar otra vez.
+      success: Se agregó %{code} a %{name}.
+      confirm: ¿Estás segura de que quieres eliminar? %{code} de %{name}?
+    source_params:
+      already_taken: "%{code} ya está siendo utilizado por <a href=%{vita_partner_path}>%{vita_partner_name}</a>."
```

Comment:
Same issue about quotes when passing parameters to HREF

Reply:
This makes me think one day we might want to add https://github.com/fnando/i18n-js to our stack. No rush; just mentioning it in case you have ideas about it.